compose.yaml

```diff
@@ -0,0 +1,22 @@
+services:
+  db:
+    hostname: postgres
+    image: "postgres"
+    volumes:
+      - data:/etc/data
+    ports:
+      - "5432:5432"
+    shm_size: 128mb
```

Comment:
why are you using `shm_size: 128` ?

Reply:
shm_size is unnecessary, it was leftover when I was copying the compose.yaml from another project.